### PR TITLE
EIP-2020: E-Money Standard Token

### PR DESCRIPTION
**Pull request by daniel-iobuilders**
The EM Token aims to enable the issuance of regulated electronic money on blockchain networks, and its practical usage in real financial applications.

### CODE REVIEW

File: EIPS/eip-2020.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
+---
+eip: 2020
+title: EM Token
+author: Julio Faura <julio@adhara.io>, Fernando Paris <fer@io.builders>, Daniel Lehrner <daniel@io.builders>
+discussions-to: https://github.com/IoBuilders/EIPs/issues/6
```
Comment (axic) [inline EIPS/eip-2020.md]: I suggest to create a discussion URL on this repo, just like you did for the other EIPs.

File: EIPS/eip-2020.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
+---
+eip: 2020
+title: EM Token
```
Comment (axic) [inline EIPS/eip-2020.md]: I suggest to choose a different title, as this is not very descriptive.

File: EIPS/eip-2020.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ The EM Token builds on Ethereum standards currently in use such as [ERC-20], but
...
+The EM Token is thus different from other tokens commonly referred to as "stable coins" in that it is designed to be issued, burnt and made available to users in a compliant manner (i.e. with full KYC and AML compliance) through a licensed vehicle (an electronic money entity, a bank, or a central bank), and in that it provides the additional functionality described above so it can be used by other smart contracts implementing more complex financial applications such as interbank payments, supply chain finance instruments, or the creation of EM-Token denominated bonds and equities with automatic delivery-vs-payment.
```
Comment (axic) [inline EIPS/eip-2020.md]: Still uses "EM Token" and "EM-Token" here.

Comment (daniel-iobuilders) [reply EIPS/eip-2020.md]: Thank you! I've changed it now and some other places where I forgot to replace it.

**Comment by daniel-iobuilders** [comment]
@axic I've made the requested changes. Please review them when you can.

**Pull request merged**